resources/visualizations/Top-Apps-By-Bandwidth-(histogram).json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Top-Apps-By-Bandwidth-(histogram) should be Top-Applications ... need to change the dashboard to look for the new version as well.